glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -277,9 +278,12 @@ open class GleanInternalAPI internal constructor () {
         return this.gleanDataDir
     }
 
-    fun collect(pingName: String) {
-        val s = LibGleanFFI.INSTANCE.glean_ping_collect(handle, pingName)!!
-        LibGleanFFI.INSTANCE.glean_str_free(s)
+    /**
+     * Collect a ping and return a string
+     */
+    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
+    internal fun collect(ping: PingType): String? {
```

Comment:
nit: do we want to follow the same naming schema as other test only functions, e.g. `testCollect`?

Reply:
Yeah -- that's a good idea.